backend/s3/s3.go

```diff
@@ -2111,6 +2120,7 @@ type Options struct {
 	SSEKMSKeyID           string               `config:"sse_kms_key_id"`
 	SSECustomerAlgorithm  string               `config:"sse_customer_algorithm"`
 	SSECustomerKey        string               `config:"sse_customer_key"`
+	SSECustomerKeyB64     string               `config:"sse_customer_key_base64"`
```

Comment:
I prefer these to match so `SSECustomerKeyBase64` even if it is a bit of a mouthful!

Reply:
done